Formula/a/asn.rb

```diff
@@ -0,0 +1,36 @@
+class Asn < Formula
+  desc "Organization lookup and server tool (ASN / IPv4 / IPv6 / Prefix / AS Path)"
+  homepage "https://github.com/nitefood/asn"
+  url "https://github.com/nitefood/asn/archive/refs/tags/v0.74.tar.gz"
+  sha256 "d51895526b1a98ae6865094aac10c1abb05b515005702f5a23ac8e1c60e159e8"
+  license "MIT"
+  head "https://github.com/nitefood/asn.git", branch: "master"
+
+  depends_on "aha"
+  depends_on "bash"
+  depends_on "coreutils"
+  depends_on "grepcidr"
+  depends_on "ipcalc"
+  depends_on "jq"
+  depends_on "mtr"
+  depends_on "nmap"
+
+  uses_from_macos "curl"
+  uses_from_macos "whois"
+
+  def install
+    bin.install "asn"
+  end
+
+  def caveats
+    <<~EOS
+      To enable the "AS path tracing" feature, MTR should be in $PATH:
+        brew link mtr
+    EOS
+  end
```

Comment:
It always is, unless people disabled it explicitly. This should not be a caveat.

Reply:
Removed